### ISSUE TITLE
Enhancement: Functionality to adjust the degree (Temperature/Entropy) of the trained dataset when generating final midi files

### ISSUE THREAD

**Issue by hpssjellis** [tensorflow/magenta#100]
A functionality to allow some variability in Magenta's ability to snap to a remembered song. As the loss becomes closer to zero Magenta tends to remember a trained song instead of merging the components of several songs.

See discussion on [google groups tensorflow magenta discuss](https://groups.google.com/a/tensorflow.org/forum/#!forum/magenta-discuss) at 

https://groups.google.com/a/tensorflow.org/forum/#!topic/magenta-discuss/SCZR9qrQUGM

I have setup a magenta fork at

https://github.com/hpssjellis/magenta

PR's are welcome.

**Comment by cghawthorne** [comment]
I think the pull request in #103 will fix this for you.
